Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,246 @@
+Feature: Test the validations done on fields from PlatformUI - text line fieldtype
+    In order to validate the text line fieldtype
+    As an Editor user
+    I need to be able to create and update content with text line fieldtypes
+
+    Background:
+       Given I am logged in as an 'Administrator' in PlatformUI
+
+    ##
+    # Validate the existence of expected fields from a field type when creating a content
+    ##
+    @javascript @common
+    Scenario: A Content of an Content Type that has a text line fieldtype must have a text field
+        Given a Content Type with an "text line" Field exists
+        When I create a content of this type
+        Then I should see a "text line" field
+
+    @javascript @common
+    Scenario: When editing  Content, the label of a text line field must have the same name than field type from the respective Content Type
```

Comment:
`When editing a Content item, the label of a Text Line field must be the corresponding Field Definition name`